tests/benchdnn/utils/cold_cache.cpp

```diff
@@ -167,9 +167,13 @@ cold_cache_t::cold_cache_t(
         auto orig_cc_mem_md = query_md(orig_mem);
 
         for (size_t i = 0; i < n_buffers_; i++) {
-            // Data will be filled/reordered, no need to prefill it.
-            cc_entry[i] = dnn_mem_t(
-                    orig_cc_mem_md, get_test_engine(), /* prefill = */ false);
+            // If there are multiple buffers, don't rely on reorders because it
+            // takes long time. Rely on a default initialization in this case.
```

Comment:
`takes long time -> takes a long time`

Reply:
Thanks for catching typos and suggestions. Applied them all.